frontend/src/component/feature/FeatureView/FeatureOverview/ManageTagsDialog/ManageBulkTagsDialog.tsx

```diff
@@ -106,6 +112,11 @@ export const ManageBulkTagsDialog: VFC<IManageBulkTagsDialogProps> = ({
         removedTags: [],
     });
 
+    const submitAndClear = () => {
+        onSubmit(payload);
+        dispatch({ type: 'reset' });
+    };
```

Comment:
maybe `submitAndReset`? Clear is related to removing also "initialValues"

Reply:
Yes, good point. I agree with this. But do you have a different word for clear, because having both is really confusing me. Maybe "resetToInitialValues" or something?